### PR TITLE
Add `ord wallet runics` command

### PR DESCRIPTION
**Pull request by ldiego08**
Add a new `ord wallet runic` command showing runic unspent outputs and balances. Solves #3729.

### CODE REVIEW

File: src/subcommand/wallet/runics.rs
Comment (ldiego08) [inline src/subcommand/wallet/runics.rs:1]: @raphjaph, does something like this satisfy #3734? It's pretty scrappy code - more interested in functionality atm. I'll polish it if green-lighted.

File: src/subcommand/wallet/runics.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+use super::*;
+
+#[derive(Serialize, Deserialize)]
+pub struct RunicUtxo {
+  pub output: OutPoint,
+  pub rune: SpacedRune,
+  pub rune_balance: u128,
+  pub amount: u64,
```
Comment (raphjaph) [inline src/subcommand/wallet/runics.rs:8]:
```suggestion
```

I know I said to add this but after thinking about it I think we should just show the runes balance and not the btc amount.

File: src/subcommand/wallet/runics.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+use super::*;
+
+#[derive(Serialize, Deserialize)]
+pub struct RunicUtxo {
+  pub output: OutPoint,
+  pub rune: SpacedRune,
+  pub rune_balance: u128,
```
Comment (raphjaph) [inline src/subcommand/wallet/runics.rs:7]:
```suggestion
  pub runes: BTreeMap<SpacedRune, Decimal>,
```
In theory an output can contain multiple runes, so let's make this a map. Have a look at `src/subcommand/wallet/balance.rs` for how to create this struct.

**Review by raphjaph — CHANGES_REQUESTED** [review]
Good start, you have the green light. Don't forget to also add an integration test :)

**Review by raphjaph — APPROVED** [review]
LGTM

**Pull request merged**